x/attribute/simulation/operations_test.go

```diff
@@ -104,7 +104,7 @@ func (suite *SimTestSuite) TestSimulateMsgUpdateAttribute() {
 	r := rand.New(s)
 	accounts := suite.getTestingAccounts(r, 3)
 	suite.app.NameKeeper.SetNameRecord(suite.ctx, "example.provenance", accounts[0].Address, false)
-	suite.app.AttributeKeeper.SetAttribute(suite.ctx, types.NewAttribute("example.provenance", accounts[1].Address.String(), types.AttributeType_String, []byte("test")), accounts[0].Address)
+	suite.app.AttributeKeeper.SetAttribute(suite.ctx, types.NewAttribute("example.provenance", accounts[1].Address.String(), types.AttributeType_String, []byte("test"), nil), accounts[0].Address)
```

Comment:
You could add a few random expiring attributes in these operations to get a better sim...